poetry.lock

```diff
@@ -4270,6 +4270,7 @@ description = "HTTP library with thread-safe connection pooling, file post, and
 optional = false
 python-versions = "!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*,>=2.7"
 groups = ["main", "dev"]
+markers = "python_version < \"3.10\""
```

Comment:
Just this silly little `markers` thing that allows Poetry to lock multiple versions of the same package is worth updating to 2.0